google/datalab/contrib/mlworkbench/commands/_ml.py

```diff
@@ -206,7 +206,7 @@ def ml(line, cell=None):
               num_workers: Dataflow number of workers. If not set, DataFlow
               service will determine the number.
```

Comment:
I don't know how textwrap.dedent is supposed to work, but should this line be indented one tab stop, such as line 209?